doc/references.md

```diff
@@ -23,8 +23,8 @@ Contents
   * describe.xml
   * commands.cfg
 
-* VMware Aria Operations Integration Library
-  * [Python](python-integration-module.md)
+* VMware Aria Operations Integration SDK Library
+  * [Python](https://pypi.org/project/vmware-aria-operations-integration-sdk-lib/)
     * Object model
       * Adapter Instance
         * Identifiers
```

Comment:
Can we update the outline to match the current structure of the integration-sdk-lib page? and possibly link within the outline the page for high-level sections if applicable

Reply:
I think I was thinking this was something other than what it actually was. I was thinking it was a link to the 'adding to an adapter' page that had been modified, and wanted to make sure that the outline matched the new titles/sections, and links were good.

I think it's good that the outline matches now, but I'm not sure that we should be linking to html pages that don't render properly.